docs/plugins/virtual_environment_docs/5-survey.md

```diff
@@ -0,0 +1,115 @@
+# Survey/Q&A
+
+This guide contains questions that were submitted to our survey on testing.  
+
+## This guide covers:   
+* [Questions](#questions)  
+    - [Testing plugins](#testing-plugins)  
+    - [Testing widgets](#testing-widgets)  
+    - [Better signal or slot documentation or functionality](#better-signal-or-slot-documentation-or-functionality)  
+    - [Creating or deleting a point or shape layer](#creating-or-deleting-a-point-or-shape-layer)  
+    - [Github tests failing](#github-tests-failing)  
+    - [Cancelling a method](#cancelling-a-method)  
+    - [Testing environments](#testing-environments)  
+* [Topics](#topics)  
+    - [npe2](#npe2)  
+    - [Widget development](#widget-development)
+  
+## Questions
+
+### Testing plugins    
+Question: What are the best practices to test a plugin with multiple sequential steps?  
+e.g. Is it ok to rely on the "results" of a test to run the next test or should they all be fully independent?  
+
+Answer:  
```

Comment:
Once we make the headings themselves questions, the answer can just be the content under that heading and we can get rid of this.

Reply:
I fixed this.